test/kuttl/multi-dcs/00-createCluster.yaml

```diff
@@ -9,7 +18,7 @@ spec:
   hardAntiAffinity: false
   deletePVC: true
   autoPilot: true
-  autoUpdateSeedList: false
+  autoUpdateSeedList: true
   dataStorageClass: local-path
   resources:
     requests: &requests
```

Comment:
whenever you see 
```
  resources:
    requests: &requests
      memory: ...
      cpu: ...
    limits: *requests
```
please replace it by
```
  resources:
    limits:
      memory: ...
      cpu: ...
```

Reply:
Okay good to know, i'm changing it right away !